config/rbac/applicationsnapshotenvironmentbinding_editor_role.yaml

```diff
@@ -0,0 +1,24 @@
+# permissions for end users to edit applicationsnapshots.
```

Comment:
to edit applicationsnapshotenvironmentbindings

Reply:
In the latest files changed for me, this still says applicationsnapshots

Reply:
The rebase did nasty things ;)